specification/attributes/attributes.mdpp

```diff
@@ -0,0 +1,10 @@
+# Attributes
+
+Attributes are requirements that apply to the billing datasets. Requirements on data content can include naming
```

Comment:
nit: I would recommend keeping paragraphs on a single line so insertions don't look like we're changing the entire paragraph, when we're just moving words to keep line lengths down.

Reply:
I thought about this too. It's a really crappy experience for anyone trying to update/maintain text in a line that goes on for ever. Do we know if there's a standard way for doing this? I see the same pros and cons listed online. Some people break down by sentence. Open to changing in the future if the group feels strongly one way or the other.